subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -2258,6 +2258,12 @@ static int lwm2m_read_handler(struct lwm2m_engine_obj_inst *obj_inst,
 			continue;
 		}
 
+		/* v1.1 allows resource instance level access */
+		if (msg->path.level == 4U &&
+			msg->path.res_inst_id != res->res_instances[i].res_inst_id) {
```

Comment:
Would be nice to align to the opening bracket, currently the alignment blends with the code.

Reply:
Fixed.